src/Stripe.net/Infrastructure/Requestor.cs

```diff
@@ -239,9 +239,13 @@ private static StripeResponse BuildResponseData(HttpResponseMessage response, st
         {
             var result = new StripeResponse
             {
-                RequestId = response.Headers.Contains("Request-Id") ? response.Headers.GetValues("Request-Id").First() : "n/a",
-                RequestDate = Convert.ToDateTime(response.Headers.GetValues("Date").First(), CultureInfo.InvariantCulture),
-                ResponseJson = responseText
+                RequestId = response.Headers.Contains("Request-Id") ?
+                    response.Headers.GetValues("Request-Id").First() :
+                    "n/a",
+                RequestDate = response.Headers.Contains("Date") ?
+                    Convert.ToDateTime(response.Headers.GetValues("Date").First(), CultureInfo.InvariantCulture) :
+                    default(DateTime),
```

Comment:
I couldn't find a way to mock response headers, so I made the `Date` header optional. I think that's better anyway.